<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playercreator</class>
 <widget class="QDialog" name="playercreator">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image : url(:/old_crackly_parchment_texture_by_Teeth_Man.png)</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>372</width>
     <height>220</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>50</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>50</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="bl2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lb1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>职业</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="lb3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>80</horstretch>
        <verstretch>30</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="name1"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cb1">
      <item>
       <property name="text">
        <string>法师</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>战士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>盗贼</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="chk1">
      <property name="text">
       <string>启用</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="name2">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cb2">
      <item>
       <property name="text">
        <string>法师</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>战士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>盗贼</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="chk2">
      <property name="text">
       <string>不可用</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="name3">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cb3">
      <item>
       <property name="text">
        <string>法师</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>战士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>盗贼</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="chk3">
      <property name="text">
       <string>不可用</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>playercreator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>playercreator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk1</sender>
   <signal>toggled(bool)</signal>
   <receiver>name1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk2</sender>
   <signal>toggled(bool)</signal>
   <receiver>name2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk3</sender>
   <signal>toggled(bool)</signal>
   <receiver>name3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
